### ISSUE TITLE
Requests with test_client raises UserWarning

### ISSUE THREAD

**Issue by vonNiklasson** [huge-success/sanic#1792]
**Describe the bug**
When doing requests with `sanic.app.test_client`, such as `sanic.app.test_client.get(...)` you get a warning (UserWarning) raised by `httpx`

Removing `verify=False` from [line 41](https://github.com/huge-success/sanic/blob/v19.12.2/sanic/testing.py#L41) in `testing.py` solves the issue but may cause other issues with SSL-certificates.

```python
try:                                                    # 39
    response = await getattr(session, method.lower())(  # 40
        url, verify=False, *args, **kwargs              # 41 <-
    )                                                   # 42
except NameError:                                       # 43
    raise Exception(response.status_code)               # 44
```

**Code snippet**
```python
from app import app

_, response = app.test_client.get("valid_project_URL")
assert response.status == 200
```

**Expected behavior**
Nothing, but instead get served with the following warning.
```
<project_root>/.venv/lib/python3.7/site-packages/httpx/client.py:234: UserWarning: Passing a 'verify' argument when making a request on a client is due to be deprecated. Instantiate a new client instead, passing any 'verify' arguments to the client itself.
"Passing a 'verify' argument when making a request on a client "
```

**Environment**
 - **OS**: macOS Catalina 10.15.3
 - **Python** version: 3.7.4
 - **Packages**:
   - `Sanic` version: 19.12.2
   - `httpx` version: 0.9.3

**Additional context**
The tests passes, but throws the warning for every single usage of `app.test_client.get(...)`.

**Comment by ashleysommer** [comment]
Fixed by https://github.com/huge-success/sanic/pull/1794

**Comment by vonNiklasson** [comment]
@ashleysommer Cheers, I saw that. Should I close this now or when a new release is published?

**Comment by Tronic** [comment]
Close now. Do note that httpx>=0.12 prevents Sanic from even starting until #1806 gets merged and released hopefully soon in 20.3. Downgrade to httpx 0.11 if this becomes an issue.

**Comment by vonNiklasson** [comment]
Cheers, closing it now then as it's resolved in #1794
